### DISCUSSION TITLE
[Support]: Frigate+ page does not show

### DISCUSSION THREAD

**Discussion by ProsjektX** [blakeblackshear/frigate#15101] [General Support]
### Describe the problem you are having

Have registered and generated the key. The key is set in the docker config (using Unraid) and is visible in the variable in the container, PLUS_API_KEY. However the plus button does not appear in the menu bar. Are there any particular ports to be opened in addition? Something else? Have followed the docs. No errors show in the log.

### Version

0.15b2

### Frigate config file

```yaml
n/a
```

### Relevant Frigate log output

```shell
n/a
```

### Relevant go2rtc log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Frigate stats

```json
n/a
```

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
n/a
```

### Object Detector

OpenVino

### Network connection

Wired

### Camera make and model

n/a

**Comment by NickM-27** [comment]
Please read the release notes, the Frigate+ page does not exist anymore in 0.15 as it has been replaced by the explore page.
